src/main/java/hudson/plugins/ec2/EC2Cloud.java

```diff
@@ -149,6 +162,7 @@ public abstract class EC2Cloud extends Cloud {
      */
     @CheckForNull
     private String credentialsId;
+
```

Comment:
Unnecessary change. Please review your own diff and check for hunks which do not contribute anything to the change, so that reviewers can focus on the hunks which do.

```suggestion
```